### PR TITLE
[zh-cn]: update the translation of Navigator: buildID property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/buildid/index.md [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
 slug: Web/API/Navigator/buildID
+l10n:
+  sourceCommit: ef75c1741b450c2331204be5563ee964ad5f4c48
 ---
 
-{{ ApiRef("HTML DOM") }}
+{{ApiRef("HTML DOM")}}{{Non-standard_Header}}
 
-返回所使用浏览器的构建标识符。现代浏览器中，这个属性返回一个固定的时间戳作为私有的计量方法，比如 Firefox 64 及以后的版本返回 `20181001000000`。
+返回浏览器的构建标识符。在现代浏览器中，此属性现在出于隐私保护措施而返回固定的时间戳，例如 Firefox 64 及更高版本中的 `20181001000000`。
```
Comment (T34-active) [inline files/zh-cn/web/api/navigator/buildid/index.md:10]:
```suggestion
返回浏览器的构建标识符。在现代浏览器中，此属性现在出于隐私保护措施而返回固定的时间戳，例如 Firefox 64 及更高版本中返回 `20181001000000`。
```

File: files/zh-cn/web/api/navigator/buildid/index.md [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
 slug: Web/API/Navigator/buildID
+l10n:
+  sourceCommit: ef75c1741b450c2331204be5563ee964ad5f4c48
 ---
 
-{{ ApiRef("HTML DOM") }}
+{{ApiRef("HTML DOM")}}{{Non-standard_Header}}
 
-返回所使用浏览器的构建标识符。现代浏览器中，这个属性返回一个固定的时间戳作为私有的计量方法，比如 Firefox 64 及以后的版本返回 `20181001000000`。
+返回浏览器的构建标识符。在现代浏览器中，此属性现在出于隐私保护措施而返回固定的时间戳，例如 Firefox 64 及更高版本中返回 `20181001000000`。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/buildid/index.md:10]:
```suggestion
返回浏览器的构建标识符。在现代浏览器中，为了保护隐私，该属性现在返回一个固定的时间戳，例如 Firefox 64 及更高版本中返回 `20181001000000`。
```

File: files/zh-cn/web/api/navigator/buildid/index.md [hunk 3]
```diff
@@ -1,26 +1,22 @@
...
+## 值
 
-```js
-buildID = navigator.buildID;
-```
-
-### 值
-
-一个字符串，用来表示当前应用的构建标识。构建 ID 的格式为：`YYYYMMDDHHMMSS`。
+一个代表应用程序构建标识符的字符串。构建 ID 的格式为 `YYYYMMDDHHMMSS`。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/buildid/index.md:14]:
```suggestion
一个表示应用程序构建标识符的字符串。构建 ID 的格式为 `YYYYMMDDHHMMSS`。
```

**Review by JasonLamv-t — COMMENTED** [review]
Almost perfect, just some nits.

**Review by JasonLamv-t — APPROVED** [review]
Thank you! @T34-active 👍🏼

**Pull request merged**